### ISSUE TITLE
Regarding use of "lazysizes" as name for WordPress plugin

### ISSUE THREAD

**Issue by 16patsle** [aFarkas/lazysizes#432]
Hello, and thanks for making such a great library!

I am currently developing a WordPress lazy load plugin using this library, and was wondering if I could name it "lazysizes" after this library. I will of course attribute the contributors of this project, and not claim I made the library myself.

I am aware I probably could just go ahead and call my plugin whatever I want, as lazysizes afaik is not trademarked, but I do not want to mislead anyone, so I am asking for permission. No need to be a jerk, you know.

So in short:
**Would you be okay with me naming my WordPress plugin "lazysizes", if I make it clear it is not a part of the official lazysizes project?**

**Comment by aFarkas** [comment]
I think so.

**Comment by 16patsle** [comment]
Great! Thank you.
